storagenode/storagenodedb/pieceexpiration.go

```diff
@@ -16,17 +16,18 @@ import (
 // ErrPieceExpiration represents errors from the piece expiration database.
 var ErrPieceExpiration = errs.Class("piece expiration error")
 
+const (
+	// PieceExpirationDBName represents the database filename.
+	PieceExpirationDBName = "piece_expiration"
+)
+
 type pieceExpirationDB struct {
-	location string
-	SQLDB
+	storageNodeSQLDB
 }
 
-// newPieceExpirationDB returns a new instance of pieceExpirationDB initialized with the specified database.
-func newPieceExpirationDB(db SQLDB, location string) *pieceExpirationDB {
-	return &pieceExpirationDB{
-		location: location,
-		SQLDB:    db,
-	}
+// newPieceExpirationDB returns a new instance of pieceExpirationDB.
+func newPieceExpirationDB() *pieceExpirationDB {
```

Comment:
I'm not sure whether the constructors are doing anything useful... might as well have `&pieceExpirationDB{}`... it would be just as clear.

Reply:
Done!